### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/font-manager/font-manager/) for Font Manager/font-manager.

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -334,10 +337,8 @@ msgid "Installable Embedding"
 msgstr "Устанавливаемое встраивание"
 
 #: lib/common/font-manager-freetype.c:703
-#, fuzzy
-#| msgid "Unknown"
 msgid "Unknown Vendor"
-msgstr "Неизвестно"
+msgstr "Неизвестный продавец"
```
Comment (PF4Public) [inline po/ru.po:341]: Is it a joke?

File: po/ru.po [hunk 2]
```diff
@@ -641,12 +642,29 @@ msgid ""
...
+"    вместе с этой программой.\n"
+"\n"
+"    Если нет, см. <http://www.gnu.org/licenses/gpl-3.0.txt>.\n"
+"    "
 
 #: src/font-manager/About.vala:67
 msgid "translator-credits"
 msgstr ""
-"PF4Public\n"
-"Александр Прокудин"
```
Comment (PF4Public) [inline po/ru.po:649]: Why?

**Review by PF4Public — COMMENTED** [review]
@JerryCasiano may I ask you to ping me before merging russian at least?

**Comment by JerryCasiano** [comment]
Of course, I can definitely ping you.

Obviously, I can't tell whether translations themselves make sense or not... So I usually just merge whatever Weblate sends over.

Might need to look into whether Weblate allowed locking other translators out of a certain language.

**Comment by PF4Public** [comment]
Ok, good to hear. I'll look into current russian translation later when I'll have time.

Small context for you:
"продавец" is a "seller". Vendors can be sellers sometimes of-course, but in the context, as it is used in font-manager, it is hardly a seller in a broader view. I wouldn't call it a nonsense-translation. Well, careless at least.
Oh, btw, could we probably rephrase this vendor? Could it be a "developer" maybe? Or any other word?

**Comment by JerryCasiano** [comment]
Oh yeah, feel free translate it to whatever makes the most sense for the audience. 

In this case, people understanding what it's referring to is more important than it being technically correct.

Vendor isn't really the most appropriate term even in English, at least IMHO, but it is the most commonly accepted since that's what was used in the font spec.

BTW, a couple strings did change in the last commit, but I think only one needs to be updated it changed from "Search Languages..." to "Search Orthographies..." for consistency.

**Comment by JerryCasiano** [comment]
So, I looked into access controls for Weblate and they are available but unfortunately only for paid plans or self hosted, not for the libre plans.

I think, if it works for you, the best way to handle this would probably be to open a new issue whenever a new release is around the corner so that contributors have a chance to test, update translations, give feedback or whatever else. In that case, you can simply submit a pull request, if needed, to overwrite any weird changes caused by Weblate pull requests.

**Comment by PF4Public** [comment]
> Vendor isn't really the most appropriate term even in English, at least IMHO

Agreed!

>but it is the most commonly accepted since that's what was used in the font spec.

Didn't know that. Vendor it is then.


> I think, if it works for you, the best way to handle this would probably be to open a new issue whenever a new release is around the corner so that contributors have a chance to test, update translations, give feedback or whatever else.

Best option indeed! Instead of pinging each PR, doing it once before a release is the best course of action. Instead of creating a new issue every time attention is needed, you could create a single issue for this purpose and open-close it accordingly. Given said that I'm now in doubt whether GitHub sends notifications on issue reopening-closing :-/

Do you have any comments on removing translator credits? Is it an error, or is this field no longer in use? If that's en error, you should probably look through files for other such modifications.

**Comment by JerryCasiano** [comment]
> Do you have any comments on removing translator credits? Is it an error, or is this field no longer in use?

That field is used in the about dialog. 

No idea why it would have been removed, it wasn't changed in the code itself, definitely an error. 

Will have to go through and manually fix those from previous commits, I guess... :-/

**Comment by PF4Public** [comment]
This proves, that no translation system is perfect :)

**Comment by JerryCasiano** [comment]
> This proves, that no translation system is perfect :)

So I had another look at this and it wasn't something done automatically by Weblate as only the russian translation seems to have been modified in this way. Looking at history it appears to have been manually removed by another translator. I'm not certain why, but Weblate flags a warning due to difference in newlines between the source string and the credits which is expected but may be why it was removed.

In any case, I've reverted the change and added a comment / note so that hopefully this does not happen again.

I think adding comments / context to any areas where translators are having issues is going to be beneficial either way.

**Pull request merged**